README.md

```diff
@@ -5,6 +5,13 @@
   - Docker
   - Java
   - Node
+  - application.yml: add this file to the ./resources folder
```

Comment:
`application.yml` already exists [here](https://github.com/milesressler/mosaic-oms/blob/main/src/main/resources/application.yml) - this should be `application-dev.yml`, which holds the configs only active with the `dev` spring profile is active